packages/amplify-cli/src/index.ts

```diff
@@ -176,15 +176,15 @@ const ensureFilePermissions = (filePath: string): void => {
 
 // This function cannot be converted to an arrow function because it uses 'this' binding
 // eslint-disable-next-line func-style
-async function sigIntHandler(this: Context): Promise<void> {
-  this.usageData.emitAbort();
+async function sigIntHandler(context: Context): Promise<void> {
+  void context.usageData.emitAbort();
```

Comment:
what does `void` do here?

Reply:
Nevermind, found it! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void

Reply:
It causes the statement to evaluate to undefined. Effectively it suppresses a promise rejection here. What route do we want to take with the usageData cases? The promises returned by usageData are not handled anywhere in the codebase. `void` was my stab at it, but I'm not sure that is the right approach.

Reply:
The intent was to fire and forget, so I think `void` makes sense.